astro/src/content/docs/get-started/core-concepts/users.mdx

```diff
@@ -71,22 +71,36 @@ These are the default fields that are available when creating a user. You can ad
   <APIField name="Tenant" required>
     The tenant to which this user belongs. This field is only displayed once multiple tenants exist in FusionAuth. When only a single tenant exists, the user will always be created in the default tenant.
   </APIField>
-  <APIField name="Skip email verification" optional>
-    If this is checked, FusionAuth will not send an email asking the user to verify their email address. This is only available if [Email verification](/docs/get-started/core-concepts/tenants#email) in the tenant is set up.
+  <APIField name="Skip verification" optional>
```

Comment:
`optional` is accepted as a parameter but is ignored in the component. We can either remove it here or mod the component to show that the field is optional.

Reply:
I think someone decided at some point that this was a more clear signal than saying nothing, since `required` (or the absence of it) is the property that actually matters. But I think this would be a broader refactoring effort that would touch a lot of files. Not sure we want to do that in a long running branch.